File: VB-Examples/05_Outlook Storage File/AddSubFolder.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Email
Imports Spire.Email.Outlook

Namespace AddSubFolder
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            Dim olf As Spire.Email.Outlook.OutlookFile = New OutlookFile("..\..\..\..\..\..\Data\Sample.pst")
            olf.RootOutlookFolder.AddFolder("NewFolder", "Spire")
            MessageBox.Show("Completed")
		End Sub
	End Class
End Namespace

File: CS-Examples/05_Outlook Storage File/AddSubFolder.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Email;
using Spire.Email.Outlook;

namespace AddSubFolder
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            Spire.Email.Outlook.OutlookFile olf = new OutlookFile(@"..\..\..\..\..\..\Data\Sample.pst");
            olf.RootOutlookFolder.AddFolder("NewFolder", "Spire");
            MessageBox.Show("Completed");
        }
    }
}